systemd/usr/libexec/miriway-session-startup

```diff
@@ -57,7 +57,8 @@
 #
 export XDG_SESSION_DESKTOP="${XDG_SESSION_DESKTOP:-Miriway}"
 export XDG_SESSION_TYPE=wayland
-VARIABLES="DESKTOP_SESSION XDG_CURRENT_DESKTOP XDG_SESSION_DESKTOP XDG_SESSION_TYPE DISPLAY WAYLAND_DISPLAY"
+export XDG_DESKTOP_PORTAL_DIR="$SNAP/usr/share/xdg-desktop-portal/portals/"
```

Comment:
As mentioned by @Conan-Kudo on Matrix, this script isn't guaranteed to execute in a Snap context.

It is probably best to create a `snap-utils` wrapper script that sets `XDG_DESKTOP_PORTAL_DIR`, runs this startup script, then runs the portal binaries

Reply:
The environment can probably just go into `snapcraft.yaml`?